src/gettingstarted/own-code/app-configuration.md

```diff
@@ -0,0 +1,106 @@
+# Import your own code
+
+## Configure application
+
+You will next need to include information that defines how you want your [application](/configuration/app-containers.md) to behave each time it is built and deployed on Platform.sh in a `.platform.app.yaml` file.
+
+```.
+├── .platform
+│   ├── routes.yaml
+│   └── services.yaml
+├── .platform.app.yaml
+└── < application code >
+```
+
+An example `.platform.app.yaml` looks like this:
+
+{% codetabs name="PHP", type="php", url="https://raw.githubusercontent.com/platformsh/template-php/master/.platform.app.yaml" -%}
+
+{%- language name="Go", type="js", url="https://raw.githubusercontent.com/platformsh/template-golang/master/.platform.app.yaml" -%}
+
+{%- language name="Python", type="py", url="https://raw.githubusercontent.com/platformsh/template-python3/master/.platform.app.yaml" -%}
+
+{% language name="Node.js", type="js", url="https://raw.githubusercontent.com/platformsh/platformsh-example-nodejs/master/.platform.app.yaml" -%}
+
+{%- endcodetabs %}
+
+
+The `.platform.appl.yaml` file is extremely flexible, and can contain many lines with very fine-grained control over your application. At the very least, Platform.sh requires four principle attributes in this file to control your builds:
+
+* `name`: The [name of your application](/configuration/app/name.md) container does not have to be the same as your project name, and in most single application cases you can simply name it `app`. You should notice in the next step, when you configure how requests are handled in `.platform/routes.yaml` that `name` is reused there, and it is important that they are the same.
+
+    > **Note**  If you are trying to to deploy [microservices](/configuration/app/multi-app.md#example-of-a-micro-service-multi-app), the only constraint is that each of these application names must be unique.
+
+* `type`: The [type](/configuration/app/type.md) attribute in `.platform.app.yaml` sets the container base image for the application, and sets the primary language. In general, `type` should have the form
+
+  ```yaml
+  type: <runtime>:<version>
+  ```
```

Comment:
Why is this a codeblock callout, but the one on the services page is not?  Let's be consistent.  (I don't really care which direction.)